sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -156,4 +159,32 @@ export class EducationProgramController {
     };
     return this.programService.saveEducationProgram(saveProgramPaylod);
   }
+
+  /**
+   * This retuns only the subset of the EducationProgram to get
+   * the complete EducationProgram DTO use the @Get(":id") method
+   * @param programId
+   * @returns
+   */
+  @Get("/program/:programId")
```

Comment:
Can this be renamed to to ":id/summary"? In this way the full endpoint would be institution/education-program/:id/summary.

Reply:
okay, I was out of ideas :)

Reply:
I believe that in this way it is going to provide a better idea that this is a "subset" of the other one that is returning all the data "institution/education-program/:id"

Reply:
done